installation/scripts/testing.sh

```diff
@@ -34,56 +39,56 @@ echo "----------------------------"
 
 echo "- Testing Core components..."
 # timeout set to 10 minutes
-helm test core --timeout 600
+helm $KUBE_CONTEXT_ARG test core --timeout 600
```

Comment:
Let's stay on safe side of the code ;) Use double-quotes and curly braces like this: "${KUBE_CONFIG_ARG}"